test/validation/api/timer/timer.c

```diff
@@ -3270,13 +3233,39 @@ int main(int argc, char *argv[])
 	if (odp_cunit_parse_options(argc, argv))
 		return -1;
 
+	if (global_init())
+		return -1;
+
 	odp_cunit_register_global_init(timer_global_init);
 	odp_cunit_register_global_term(timer_global_term);
 
-	int ret = odp_cunit_register(timer_suites);
+	if (odp_cunit_register(timer_general_suites))
+		goto fail;
 
-	if (ret == 0)
-		ret = odp_cunit_run();
+	if (odp_cunit_run())
+		goto fail;
+
+	for (clk_src = ODP_CLOCK_SRC_0; clk_src < ODP_CLOCK_NUM_SRC; clk_src++) {
+		odp_timer_capability_t capa;
+
```

Comment:
Since we are running entire suite multiple times, a print with clock source ID would help to read the log when something failes.